plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -597,18 +597,22 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 		}
     </#if>
 
-	<#if data.waterMob>
-	@Override public boolean canBreatheUnderwater() {
-    	return true;
-    }
+	<#if data.breatheUnderwater?? && (hasProcedure(data.breatheUnderwater) || data.breatheUnderwater.getFixedValue())>
+		@Override public boolean canBreatheUnderwater() {
+			return <@procedureOBJToConditionCode data.breatheUnderwater true false/>;
+		}
+    </#if>

```

Comment:
The indentation here is a bit strange, seems like a mix of tabs and spaces.

I also prefer if simple blocks of if like those are not indented one tab additional after FTL if

This applies to both versions

Reply:
Indentations are still not like (most) other ifs in livingentity.java.ftl

![image](https://github.com/MCreator/MCreator/assets/16374228/cfc0088f-52c9-41fe-ace8-4a4afa16c15c)

Reply:
Oh ok, I thought you were talking about the if statements only, my bad.

Reply:
No problem, was not clear enough from my side :)